### DISCUSSION TITLE
IKOHS (by CREATE) ceiling fan discussions

### DISCUSSION THREAD

**Discussion by velzend** [velzend/create_ikohs_fan#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by henkjanvries** [comment]
Hi @velzend , ik was op zoek naar info over deze fan met tuya en zag jou github, op aanwijzing van de ha fb group. Hoe bevalt het tot nu toe? Ik ik las 2 punten waar ik twijfels bij heb, lichtsterkte: kan je helemaal niks, en besturing niet asynchroon: heb je dit in HA kunnen oplossen?

hoor graag van je!

**Reply by velzend** [reply]
Hi @henkjanvries, het besturen met Home Assistent gaat zonder problemen en vrijwel direct. In de slaapkamer heb ik bij de deur een ZigBee 8v (4x IO) switch (Sunricher) waar ik o.a. de ventilator en de LED lamp van de ventilator mee aan stuur. De LED lamp gebruiken we minimaal. De afstandsbediening gebruiken we ook, zijn er eigenlijk een soort van aan gewend dat de state niet altijd klopt.

Met de 8v kunnen we ook de ZigBee lampen op de nachtkastjes aan- en uitzetten en dimmen. En de downlights boven de kleding kast en spiegel individueel aan- en uitzetten en dimmen. 

Voor de lampen op de nachtkastjes ligt er ook nog een Philips Hue dimmer switch op de bed rand. Of alles met de Home Assistant app/ web GUI.

Over de lichtsterkte de minimale waarde is gewoon nog erg fel. En op maximale lichtsterkte is het net een bouw lamp. De licht kleur kan niet echt warm wit. 

Nu kan dit natuurlijk op diverse wijze opgelost worden, ik had er alleen iets meer van verwacht.
Ik had gehoopt dat de lichtsterkte te regelen zou zijn van bijna uit tot volledig met veel detail (veel kleine stappen). Ook de licht kleur warmer. 

Maar goed, de ventilator zelf staat bijna elke dag aan... De ventilator is qua comfort echt heel fijn apparaat en super stil... Je hoort eigenlijk alleen de wind verplaatsing. 

Hoe stuur jij de ventilator aan heb je nog leuke tips?

**Reply by henkjanvries** [reply]
Hi Dennis, ik heb momenteel geen ventilator. Ik ben op zoek. Ik had wel modellen let digitale aansturing en ifan03 gekocht. Maar dat hwb ik weer teruggestuurd omdat ik niet wist hoe ik de ifan kon integreren.
Deze ventilatoren vind ik heel interessant. Het liefst in elke slaapkamer, weet alleen niet of ik de verlichting punten voldoet aan mn eisen. Zeker voor de kinderslaapkamers. 

Zou je het opnieuw doen of toch voor een alternatief gaan?

**Reply by henkjanvries** [reply]
Hi @velzend net mijn ikohs binnengekregen. Terwijl ik aan het neuzen was kwam ik deze thread tegen. Ik denk dat je ook een ifan03 zou kunnen toevoegen aan de behuizing
https://community.home-assistant.io/t/replacing-rf-ceiling-fan-receiver-with-sonoff-ifan03/187994

heb je hier wel eens over nagedacht?

**Comment by yordivd** [comment]
Hey @velzend, ik heb ook deze plafondventilator en ik kan bevestigen dat ik net dezelfde problemen heb.
Status tussen remote en app is niet in sync + lichtwarmte veranderd als je te snel aan/uit/aan/uit doet.

Voorlopig nog niet met HA of dergelijke gekoppeld.

**Comment by rahimremtulla** [comment]
Hello,

I bought the CREATE IKOHS Windcalm DC ceiling fan last year. It is the RF model as the WiFi model was not available then. I have recently spoken to IKOHS who have said that there is WiFi module available for the fan but they are not currently shipping it to the UK. I was wondering if it is worth waiting on this or go for something like Broadlink RM4 Pro instead?

Any thoughts?

**Reply by velzend** [reply]
Hi Rahim, 

Cool that you found my Git repository. I was not yet aware of the CREATE IKOHS fan support by the Broadlink RM4 Pro. You can get those quite cheap, and the learning mode is compatible with the IKOHS remote according to some fora. 

I can't confirm though... I have a couple 433MHz transceivers somewhere, in used them back in the days I was working on the Arduino KAKU project and annoy the neighborhood by random replays... Ghehe.... Anyhow, I think this Broadlink device looks promising to have control over the fan without the remote. The only thing you won't (also) have is a good state if both are used. 

But, and here is becomes interesting... If you are able to change the rolling code of solely the remote or whatever of the remote and fan uses as unique identifier, and keep the fan's unique identifier. You may be able to relay the remote to the Broadlink and keep the state there. If the Broadlink can also listen to RF patterns and implement some automations. 

I also got questions if some other fan controllers may work... I don't mind testing stuff but finding some spare time and my wife's permission to take the fan from the ceiling / apart ....

**Comment by TheRikkert** [comment]
Hey Velzend, 

Ik probeer de IKOHS fan met een homey te integreren om zo het licht via een normale licht schakelaar ook nog aan te doen (tuimelschakelaar, dus state boeit niet zo). Echter ik loop vast op het punt dat ik van de "getting local keys". Hiervoor zeggen ze dat het device gekoppeld moet zijn in Tuya developer via de Smart life app. Ik krijg de Ikohs fan echter alleen gekoppeld aan de create app. Hoe gaat dit en zijn werk en hoe is het jou gelukt?

**Comment by hoeni** [comment]
Hey, nice to find this repo here 👍

Last year I bought a Windcalm DC without Wi-Fi, which was not available at that time, unfortunately. I decided to go for the version without lights, as I have hue everywhere. I really like the flat, simple design and the quiet operation of the fan.

However, I tried to contact support by multiple means and asked if it is possible to buy the WiFi-controller separately to "upgrade" my fan to the Wi-Fi variant, but have not seen any response.

I decided to try hacking the remote control instead. Currently, I'm waiting for my SDR-USB-Stick and the 433MHz sender shipping from China, hoping I can decode the commands and send them via MQTT with an ESP8266 or ESP32. However, I've never done wireless reverse engineering so far, so I have no idea how this will work out.

But it's good to hear that there wouldn't be any status information anyway with the Wi-Fi version :-)

I really like the idea of having a custom build DIY controller for it, but that's far out of my technical knowledge (I'm more kind of a software guy ;-) )

PS: I would love to keep that exchange here in English as I do not understand Dutch even a bit ;-)

**Comment by rahimremtulla** [comment]
My fan is going great for a couple of years now. I did send them an E-mail earlier this year and they did response with in regards to the WiFi module. They quoted EUR 25.00 for the module. I decided not get this and bought a Broadlink RM4 Pro universal remote instead. The advantage with this was that I could easily integrate it within my Home Assistant and Amazon Echo smart home set-ups.

I have just ordered another Windcalm DC fan for my living room and chose the non-WiFi option again!

**Reply by hoeni** [reply]
That's a good Idea, especially a) with the moderate price tag of the Broadlink and b) now knowing, there is no working status feedback on the "official" Wi-Fi version anyway.

**Reply by nielsvanbaal** [reply]
How does this work? Does it send ir? My original remote isn't ir far as I know. I have the WiFi option but it's far from working right with homeassistant.

**Reply by hoeni** [reply]
Thinking about it, there is only a way to increase and decrease the fan speed, but no way to set it to a specific speed, right?

**Reply by ScratMan** [reply]
It seems to be possible to set the speed directly on the "disconnected" version. I ordered three WiFi fans via Amazon and received the wrong controllers, with two remotes per fan : the standard thin one with only speed +/- buttons, and a square one with 6 speed buttons instead.

https://user-images.githubusercontent.com/971926/261833726-4e47dad3-4a80-4d88-abfd-7190d8aeeb99.jpg

**Comment by henkjanvries** [comment]
Im using it currently with the tuya integration.. the only tjing it wont
allow is turn off of the fan. Next to no data sync of status, but as
everything in the room is managed by HA i only jeed to turn off the fan. I
was curious of localtuya might have more possibilities

Ive got to say, overall works pretty good

Op vr 19 aug. 2022 om 12:39 PM schreef nielsvanbaal <
***@***.***>

**Comment by henkjanvries** [comment]
I think it just works in 25% jumps. But back to 0% wont turn the fan off..

Op vr 19 aug. 2022 om 12:58 PM schreef Tobias Henöckl <
***@***.***>

**Reply by hoeni** [reply]
That's sad, as I was hoping to be able to set it to a certain speed. This would be an argument *for* the WiFi-Version.

**Comment by carloda** [comment]
Hi, I have the IKOHS windcalm model again (by create) once integrated in LocalTuya a strange thing happens I don't see the ID62 which should be the parameter to adjust the speed, is there any remedy?

![2023-06-22 20_04_50-Impostazioni – Home Assistant](https://github.com/velzend/create_ikohs_fan/assets/35763669/a61a9a77-4f7e-4e77-a5e6-faa482a5d4ef)

**Comment by theoneill** [comment]
HI ! I am trying to repair a create ceiling fan controller : The motor control is OK but the led stopped working. After testing the triac which is used to deliver the power to the led and switch it on/off, it seems to be OK (when I shunt the triac, the light will turn on). So, I would like to check/replace the component controlling the triac i.e this one (cf. red arrow in the picture below):

![image](https://github.com/velzend/create_ikohs_fan/assets/32323500/5f30b25b-a1cb-460a-b8de-13b592959ed9)

Unfortunately, there is nothing readable on my opto triac : There is a kind of varnish but no reference written below (or it just went away with the varnish). Would someone be able to read a component ref on his circuit ? Or suggest with which component I could replace it ? I was thinking about MOC3021, MOC3023...

Thanks !

**Comment by BennySp98** [comment]
Hey how did you connect the Fan with the Tuya App? It only seems to work for me with their own CREATE App. Can't Figure out how to Reset the Fan

**Comment by jtwild** [comment]
Hi @velzend ,

Nice documentation, thanks. 
You mention you are able to adjust the brightness level of the LED's, but I don't have this option. Which DPID do you have available for switching the brightness? Mine just has on/off, and some (non-functioning) work_mode and color_temp and scene_data

**Comment by gonzalezcalleja** [comment]
Hi

where can i buy only the remote controller for DC fans? Thanks!
